### PR TITLE
NIFI-6106 Add tests for commons utilities & fix FormatUtils bug

### PR DESCRIPTION
**Pull request by freddytuxworth**
I analyzed this project and noticed some methods lacking tests in these two utilities classes. I've written some tests using [Diffblue](https://www.diffblue.com/) [Cover](https://www.diffblue.com/datasheet) which should hopefully help you to detect any regressions caused by future code changes. 

I also corrected a small bug in `FormatUtils` which caused the `formatNanos` method to incorrectly format multiples of 1000000 or 1000000000 to `"0 nanos"`.

Please let me know if there are any other areas of your project which you would like tests written for, I would be more than happy to help.

### CODE REVIEW

File: nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package org.apache.nifi.util;
+
+import static org.junit.Assert.*;
```
Comment (mattyb149) [inline nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java]: Looks good, thanks for the fix and additional tests! This line is a CheckStyle violation though, need to replace the star import with individual ones, please and thanks!

Comment (freddytuxworth) [reply nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java]: Just fixed this, build should now pass.

File: nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
+package org.apache.nifi.util;
```
Comment (mattyb149) [inline nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java]: Needs the Apache License header, see the other file you worked on (TestFormatUtils.java) for an example (but almost any file in the codebase should have it)

Comment (freddytuxworth) [reply nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java]: Added the license header.

**Comment by mattyb149** [comment]
+1 LGTM, Travis is happy so I'm happy :) Thanks for the nanos fix and the extra tests! Merging to master

**Pull request closed without merging**